go/client/cmd_status.go

```diff
@@ -92,7 +92,7 @@ func (c *CmdStatus) outputJSON(status *keybase1.FullStatus) error {
 func (c *CmdStatus) outputTerminal(status *keybase1.FullStatus) error {
 	extStatus := status.ExtStatus
 	dui := c.G().UI.GetDumbOutputUI()
-	dui.Printf("Username:      %s\n", status.CurStatus.User.Username)
```

Comment:
@jzila this was the crasher for kbfs-server tests

Reply:
Thanks!